pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -560,10 +560,11 @@ class Build {
                 def target = "testList TESTLIST=disabled.criu_pingPerf_testCreateRestoreImageAndPushToRegistry,disabled.criu-portable-checkpoint_test,disabled.criu-ubi-portable-checkpoint_test"
                 def jobParamsList = [
                     'x86-64_linux' : [
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.ubuntu.22&&hw.arch.x86.broadwell'],
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.rhel.8&&hw.arch.x86.broadwell'],
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.rhel.8&&hw.arch.x86.amd'],
-                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.os.rhel.8&&hw.arch.x86.skylake']
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.ubuntu.22&&hw.arch.x86.broadwell'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.ubuntu.22&&hw.arch.x86.amd'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.rhel.8&&hw.arch.x86.broadwell'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.rhel.8&&hw.arch.x86.amd'],
+                        ['LABEL_ADDITION' : 'sw.tool.podman&&sw.tool.container.criu&&sw.os.rhel.8&&hw.arch.x86.skylake']
```

Comment:
Should we refactor the repeated labels into a variable?

Reply:
Yes, we can, but this code will be removed eventually. The plan is to move the logic to the test pipeline. See https://github.com/adoptium/aqa-tests/issues/4465